packages/eslint-plugin-pf-codemods/lib/rules/v5/nav-warn-flyouts-now-inline.js

```diff
@@ -0,0 +1,33 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8628
+module.exports = {
+  meta: {},
+  create: function (context) {
+    const imports = getPackageImports(context, "@patternfly/react-core").filter(
+      (specifier) => specifier.imported.name === "Nav"
+    );
+
+    return imports === 0
```

Comment:
Just noticed, but this should be:

```suggestion
    return imports.length === 0
```